robottelo/cli/factory.py

```diff
@@ -452,6 +452,27 @@ def make_partition_table(options=None):
     return create_object(PartitionTable, args, options)
 
 
+def make_proxy(self, options=None):
```

Comment:
None of the helpers should be going into this module. Infact all the helpers in this module would be moved under `cli_factory` object/class. The plan is to remove the entire `cli/factory.py` in this PR itself!

Reply:
Move this under cli_factory.